app/src/main/java/com/blaizmiko/popcornapp/models/Movie/SpokenLanguage.java

```diff
@@ -0,0 +1,26 @@
+package com.blaizmiko.popcornapp.models.Movie;
+
+import com.google.gson.annotations.SerializedName;
+
+public class SpokenLanguage {
```

Comment:
Think no.
But if you asked, I think we dont need, not only this, but more fields in Movie. Just think that its better to add all, mb it would be useful in future